components/aws/sagemaker/tests/integration_tests/utils/sagemaker_utils.py

```diff
@@ -1,6 +1,33 @@
+import pytest
+from botocore.exceptions import WaiterError
+
+
 def describe_training_job(client, training_job_name):
     return client.describe_training_job(TrainingJobName=training_job_name)
 
 
 def describe_model(client, model_name):
     return client.describe_model(ModelName=model_name)
+
+
+def describe_endpoint(client, endpoint_name):
+    return client.describe_endpoint(EndpointName=endpoint_name)
```

Comment:
what happens if `describe_endpoint` generates an exception ?

Reply:
removed all exception handling since test has to fail anyways if any of these calls fail